docs/organizations/overview.mdx

```diff
@@ -172,3 +172,11 @@ If the prebuilt components don't meet your specific needs or if you require more
 - [Creating an infinite paginated list of memberships](/docs/organizations/viewing-memberships)
 - [Inviting users to an organization](/docs/organizations/inviting-users), which also includes code for creating a custom list of invitations
 - [Managing memberships](/docs/organizations/managing-roles), which includes code for updating and deleting a user's membership, for inviting a user, and for creating a custom list of memberships, invitations, and requests
+
+## Invalid states
+
+Organizations can enter invalid states under certain conditions. Here are the common scenarios you should be aware of:
```

Comment:
```suggestion
Organizations can enter invalid states under certain conditions.
```

we can either fully omit a second sentence, or have it like this:

```suggestion
Organizations can enter invalid states under certain conditions. See the following sections for more information.
```

I prefer the first.